### ISSUE TITLE
Ignition: add force option to node creation

### ISSUE THREAD

**Issue by dgonyeo** [coreos/bugs#2311]
# Issue Report #

### Environment ###

All

### Desired Feature ###

It can be desirable to create a filesystem node at a location where one already exists. As an example, symlinking `/etc/localtime` to something in `/usr/share/zoneinfo`.

A `force` flag could be added to the node object in the schema, which would result in Ignition deleting anything at the path before creating the desired node.

### Other Information ###
```
[    9.355226] ignition[416]: files: createFilesystemsFiles: createFiles: op(a): [started]  writing link "/etc/localtime" -> "/usr/share/zoneinfo/US/Arizona"
[    9.360164] systemd[1]: ignition-files.service: Failed with result 'exit-code'.
[    9.364136] ignition[416]: files: createFilesystemsFiles: createFiles: op(a): [failed]   writing link "/etc/localtime" -> "/usr/share/zoneinfo/US/Arizona": symlink /usr/share/zoneinfo/US/Arizona /sysroot/etc/localtime: file exists
```

**Comment by ethernetdan** [comment]
Example of erroring Ignition config:
```json
{
  "ignition": { "version": "2.1.0" },
  "storage": {
    "links": [{
      "filesystem": "root",
      "path": "/etc/localtime",
      "target": "/usr/share/zoneinfo/US/Pacific"
    }]
  }
}
```

**Comment by felixbuenemann** [comment]
I ran into the same problem, so until the fixes from coreos/ignition#494 land in the stable channel I'm using the following run-one systemd unit as a workaround:

```yaml
systemd:
  units:
    - name: set-default-timezone.service
      enabled: true
      contents: |
        [Unit]
        Description=Set Default System Time Zone
        ConditionFirstBoot=yes

        [Service]
        Type=oneshot
        ExecStart=/usr/bin/timedatectl set-timezone Europe/Berlin

        [Install]
        WantedBy=multi-user.target
```
